lib/stacker/stack.rb

```diff
@@ -166,7 +166,7 @@ def report_status
         failure_event = client.events.enum(limit: 30).find do |event|
           event.resource_status =~ /_FAILED$/
         end
-        failure_reason = failure_event.resource_status_reason
+        failure_reason = failure_event.nil ? "Unknown failure reason" : failure_event.resource_status_reason
```

Comment:
Just wondering if it should be `failure_event.nil? ? "Unknown" ...`. I suspect `.nil` will fail with `NoMethodError: undefined method 'nil' for...`